src/components/Form/Input.js

```diff
@@ -18,14 +18,34 @@ const Input = (props) => {
     iconRight,
     id,
     label,
+    placeholder,
+    readOnly,
     required,
     showSpinner,
+    value,
     ...rest,
   } = props;
 
   const hasIconLeft = !!iconLeft || (error && errorIcon);
   const hasIconRight = !!iconRight || !!showSpinner;
 
+  const renderInputRaw =
+    (<InputRaw
+      error={error}
+      errorIcon={errorIcon}
+      iconLeft={iconLeft}
+      iconRight={iconRight}
+      id={id}
+      label={label}
+      placeholder={placeholder}
+      readOnly={readOnly ? true : null}
```

Comment:
you can check for `readOnly` if you null in the default props, no need for a ternary